hyperkit.1

```diff
@@ -275,18 +275,18 @@ Disable MPtable generation.
 .Pp
 Firmware/bootrom configuration.
 .Bl -tag -width 10n
-.It kexec , Ns Pa kernel , Ns Oo Pa ramdisk Ns Oc , Ns Oo Pa cmdline Oc
+.It kexec , Ns Pa kernel Ns Oo , Ns Pa ramdisk Ns Oc Ns Oo , Ns Pa cmdline Oc
 Launch
 .Ar kernel
 using the Linux kexec protocol.
-.It fbsd , Ns Pa userboot , Ns Pa bootvolume , Ns Pa kernelenv
+.It fbsd , Ns Pa userboot , Ns Pa bootvolume Ns Oo , Ns Pa kernelenv Ns Oc
 Boot using the fbsd protocol.
-.It bootrom , Ns Pa path , Ns ,
+.It bootrom , Ns Pa path
 Boot using
 .Ar path
 as a firmware image. Allows booting a UEFI/tianocore binary, such as
 the one from bhyve (source: <https://github.com/freebsd/uefi-edk2.git>; binary <https://people.freebsd.org/~grehan/bhyve_uefi/>).
-.It multiboot , Ns Pa kernel , Ns Oo Pa module Ns Oo ;cmdline Ns Oc Ns Oo :module Ns Oo ;cmdline Ns Oc Ns Oc Ns ... Oc , Ns Oo Pa cmdline Ns Oc
+.It multiboot , Ns Pa kernel Ns Oo , Ns Pa module Ns Oo ;cmdline Ns Oc Ns Oo :module Ns Oo ;cmdline Ns Oc Ns Oc Ns ... Oc Ns Oo , Ns Pa cmdline Ns Oc
```

Comment:
I like the second one. While it is a bit wordy it does clearly state that the commas may be omitted in some cases.